src/routes/rss.ts

```diff
@@ -90,7 +91,9 @@ const generateRSS =
         }),
       );
       return res.type('application/rss+xml').send(feed.xml());
-    } catch (err) {
+    } catch (originalError) {
+      const err = originalError as TypeORMQueryFailedError;
+
```

Comment:
I really wish we could set type inside `catch` statement :triggered:

Reply:
Yep really sad :(